### PR TITLE
[jsk_perception] Add label_array_to_mask_image

### PR DESCRIPTION
**Pull request by knorth55**
@iory 
I found these commits not merged to `master` branch from `taichiH/master`.

I use `Useful forks` to find these unmerged or non-PR forks.
https://useful-forks.github.io/?repo=jsk-ros-pkg%2Fjsk_recognition

### CODE REVIEW

**Comment by iory** [comment]
That's an awesome function of github! Thanks!

**Review by k-okada — CHANGES_REQUESTED** [review]
nice, please add documentation @taichiH, @iory

**Comment by iory** [comment]
I sent a PR to add docs and a test.
https://github.com/knorth55/jsk_recognition/pull/4
Please check it.

File: jsk_perception/test/label_array_to_mask_image.test [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<launch>
+
+  <include file="$(find jsk_perception)/sample/sample_label_array_to_mask_image.launch">
+    <arg name="gui" value="false" />
+    <arg name="queue_size" value="100" />
```
Comment (taichiH) [inline jsk_perception/test/label_array_to_mask_image.test:5]: @iory 

Is it better to set "gpu" arg to -1?
```
<arg name="gpu" value="-1"/>
```

It seems test failes becaues there is no gpu environment.
https://github.com/jsk-ros-pkg/jsk_recognition/pull/2624/checks?check_run_id=3532914923#step:5:17104

Comment (iory) [reply jsk_perception/test/label_array_to_mask_image.test:5]: Thank you for your comment!
https://github.com/knorth55/jsk_recognition/pull/5
I sent a PR to fix this test. 

@knorth55 Please check it.

**Comment by k-okada** [comment]
```
021-09-09T07:13:37.6721772Z Full test results for 'jsk_perception/test_results/jsk_perception/rosunit-test_label_array_to_mask_image.xml'
2021-09-09T07:13:37.6723048Z -------------------------------------------------
2021-09-09T07:13:37.6723892Z <?xml version="1.0" encoding="utf-8"?>
2021-09-09T07:13:37.6724931Z <testsuite errors="0" failures="1" name="unittest.suite.TestSuite" tests="1" time="30.008">
2021-09-09T07:13:37.6726284Z   <testcase classname="__main__.TestTopicPublished" name="test_published" time="30.0077">
2021-09-09T07:13:37.6727659Z     <failure type="AssertionError">Topic [/label_array_to_mask_image/output] is not published
2021-09-09T07:13:37.6728876Z   File "/usr/lib/python2.7/unittest/case.py", line 329, in run
2021-09-09T07:13:37.6729567Z     testMethod()
2021-09-09T07:13:37.6730436Z   File "/opt/ros/indigo/share/jsk_tools/src/test_topic_published.py", line 96, in test_published
2021-09-09T07:13:37.6731353Z     checker.topic_name)
2021-09-09T07:13:37.6732151Z   File "/usr/lib/python2.7/unittest/case.py", line 422, in assertTrue
2021-09-09T07:13:37.6733078Z     raise self.failureException(msg)
2021-09-09T07:13:37.6733748Z     </failure>
2021-09-09T07:13:37.6734256Z   </testcase>
2021-09-09T07:13:37.6735058Z   <system-out><![CDATA[]]></system-out>
2021-09-09T07:13:37.6735969Z   <system-err><![CDATA[]]></system-err>
2021-09-09T07:13:37.6736597Z </testsuite>
2021-09-09T07:13:37.6736916Z 
2021-09-09T07:13:37.6737671Z
```
still failing?

**Comment by iory** [comment]
Fixed error.
All required tests passed.

**Pull request merged**